### PR TITLE
Disable requiring email for bots

### LINKED ISSUE #3818: Email validation for new bot user not required
Running #develop branch 0.35.0+

When creating user from admin interface, you must provide an email, or get the error "The field Email is required". Is this actually necessary for bot users though, do bots get notifications?

I notice Rocket.Cat has no email, so its obviously not breaking anything. I'd like to avoid having to invent arbitrary variations on email addresses for my small army of Hubots.

### PR DESCRIPTION
**Pull request by timkinnane**
@RocketChat/core 

Closes #3818, related to #3679 

See [issue notes](https://github.com/RocketChat/Rocket.Chat/issues/3818#issuecomment-234171509) for more details.

### CODE REVIEW

File: packages/rocketchat-ui-flextab/flex-tab/tabs/userEdit.coffee [hunk 1]
```diff
@@ -29,6 +29,19 @@ Template.userEdit.events
 		e.preventDefault()
 		$('#password').val(Random.id())
 
+	# set and disable password change, welcome and verified fields when bot
+	'change #role': (e) ->
+		isBot = $(e.target).val() is 'bot'
+		$ '#verified'
+		  .prop 'disabled', isBot
```
Comment (sampaiodiego) [inline packages/rocketchat-ui-flextab/flex-tab/tabs/userEdit.coffee]: there are 2 space before `.prop` instead of a `tab` =)

**Comment by sampaiodiego** [comment]
hi @timkinnane .. thanks for the fix =)

I tried this out and I didn't like the behaviour you introduced: every time I choose a role other than `bot` it is `checking` the fields: _verified_, _require password change_ and _send welcome email_.

what you think just disabling them when the `bot` role is selected and just enabling them otherwise?

**Comment by timkinnane** [comment]
Yeah, you're right @sampaiodiego - I kinda did that in a hurry, thought that it wasn't ideal but didn't have time to follow up. The reason I changed the checks is because there's no clear visual state for disabled inputs. From user's perspective they can't tell if the field is disabled or if clicks just stopped working. I guess the responsible thing to do is close this now and resubmit when I've had time to work some UI updates for disabled fields so I don't have to change selections.

**Comment by sampaiodiego** [comment]
ok @timkinnane .. thanks again, looking forward for your new changes..

**Pull request closed without merging**